common/src/main/java/com/wynntils/mc/mixin/MultiPlayerGameModeMixin.java

```diff
@@ -90,4 +90,15 @@ private void attack(Player player, Entity target, CallbackInfo ci) {
             ci.cancel();
         }
     }
+
+    @Inject(
+            method = "ensureHasSentCarriedItem",
+            at =
+                    @At(
+                            value = "INVOKE",
+                            target =
+                                    "Lnet/minecraft/client/multiplayer/ClientPacketListener;send(Lnet/minecraft/network/protocol/Packet;)V"))
+    private void ensureHasSentCarriedItem(CallbackInfo ci) {
```

Comment:
We should note this here, for future us: "As of 1.19.3, this seems to be the only method which sends carried item update packets to the server. Please look into this and confirm this is still the case, in future versions."